CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 v2.3.0
 
+* Added support for polymorphic models via `django-polymorphic`
```

Comment:
My sense is that we should make the documentation clear that `django-polymorphic` is the only *tested* polymorphic library (thereby implying *supported* library). As far as the CHANGELOG is concerned, "via `django-polymorphic`" can be deleted.